File: FoodDeliveryAPI/FoodDeliveryAPI.Domain/DTO/Token.cs
namespace FoodDeliveryAPI.Domain.DTO
{
    public class Token
    {
        public string authToken {  get; set; }
        public Guid userId { get; set; }
        public DateTime IssuedAt { get; set; }
        public DateTime ExpireAt { get; set; }
    }
}

File: FoodDeliveyAPI.Frontend/FoodDeliveryAPP/src/app/models/Token.ts
export interface Token {
  authToken: string;
  userId: string;
  issuedAt: Date;
  expireAt: Date;

  
}


// constructor(
//   authToken: string,
//   userId: string,
//   issuedAt: Date,
//   expireAt: Date
// ) {
//   this.authToken = authToken;
//   this.userId = userId;
//   this.issuedAt = issuedAt;
//   this.expireAt = expireAt;
// }